catalog.html

```diff
@@ -0,0 +1,171 @@
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="index.html">
+          <img src="" alt="Логотип Седоны">
+        </a>
+        <ul>
+          <li>
+            <a href="">Информация</a>
+          </li>
+          <li>
+            <a href="">Фото и видео</a>
+          </li>
+          <li>
+            <a href="">Карта штата</a>
+          </li>
+          <li>
+            <a href="">Гостиницы</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+    <section class="filter">
+      <form action="">
+        <fieldset>
+          <legend>Инфраструктура:</legend>
+          <ul>
+            <li>
+              <label for="check-1">
+                <input type="checkbox" checked name="check-1" id="check-1">
+                <span>Бассейн</span>
+              </label>
+            </li>
+            <li>
+              <label for="check-2">
+                <input type="checkbox" name="check-2" id="check-2">
+                <span>Парковка</span>
+              </label>
+            </li>
+             <li>
+              <label for="check-3">
+                <input type="checkbox" name="check-3" id="check-3">
+                <span>Wi-Fi</span>
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+         <fieldset>
+          <legend>Тип жилья:</legend>
+          <ul>
+            <li>
+              <label for="check-4">
+                <input type="checkbox" checked name="check-4" id="check-4">
+                <span>Гостиница</span>
+              </label>
+            </li>
+            <li>
+              <label for="check-5">
+                <input type="checkbox" checked name="check-5" id="check-5">
+                <span>Мотель</span>
+              </label>
+            </li>
+             <li>
+              <label for="check-6">
+                <input type="checkbox" checked name="check-6" id="check-6">
+                <span>Апартаменты</span>
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+        <fieldset>
+          <legend>Стоимость в сутки (Р):</legend>
+          <input type="range">
+          <button type="submit">Показать</button>
+        </fieldset>
+      </form>
+      </section>
+      <section class="content">
+        <div class="sort-top">
+          <h2>Найдено: 3</h2>
+          <ul>
+            <li>Сортировка:</li>
+            <li class="active">По цене</li>
+            <li><a href="#!">По типу</a></li>
+            <li><a href="#!">По рейтингу</a></li>
+          </ul>
+          <ul>
+            <li class="active">По убыванию</li>
+            <li><a href="#!">По возростанию</a></li>
+          </ul>
+        </div>
+        <div class="sort-rezult">
+          <article>
+            <div>
+              <img src="" alt="Amara Resort & Spa">
+              <h3>Amara Resort & Spa</h3>
```

Comment:
в стайлгаде есть состояния заголовков, так что это ссылка
```suggestion
              <h3><a href="#!">Amara Resort & Spa</a></h3>
```